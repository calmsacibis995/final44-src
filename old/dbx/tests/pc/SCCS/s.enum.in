h13842
s 00008/00000/00000
d D 5.1 90/06/25 15:36:56 bostic 1 0
c date and time created 90/06/25 15:36:56 by bostic
e
u
U
t
T
I 1
whatis Color
whatis c1
stop in P
run
where
print c
print c1, c2
quit
E 1
