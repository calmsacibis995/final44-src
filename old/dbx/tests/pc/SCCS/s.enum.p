h23134
s 00016/00000/00000
d D 5.1 90/06/25 15:36:59 bostic 1 0
c date and time created 90/06/25 15:36:59 by bostic
e
u
U
t
T
I 1
program enumtest (input, output);
type
    Color = (RED, GREEN, BLUE);

var c1, c2 : Color;

procedure P (c : Color);
begin
    writeln('c = ', ord(c):1);
end;

begin
    c1 := BLUE;
    c2 := GREEN;
    P(RED);
end.
E 1
