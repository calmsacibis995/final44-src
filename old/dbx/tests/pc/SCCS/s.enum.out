h27285
s 00011/00000/00000
d D 5.1 90/06/25 15:36:58 bostic 1 0
c date and time created 90/06/25 15:36:58 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
type Color = (RED, GREEN, BLUE);
var c1 : Color;
[1] stop in P
[1] stopped in P at line 8
    8   begin
P(c = RED), line 8 in "enum.p"
program(), line 15 in "enum.p"
main(0x1, 0x7fffe7e8, 0x7fffe7f0) at 0x105
RED 
BLUE GREEN 
E 1
